### PR TITLE
fix: requesting a static file at pl-2

### PR DESCRIPTION
**Pull request by EsadCetiner**
Fixes false positives when requesting a static file at pl-2 or higher with the ver argument. I haven't done any character whitelisting since this is just used for static files, there's nothing to attack.

### CODE REVIEW

**Comment by azurit** [comment]
@EsadCetiner How are you enforing that this exclusion is used only for static files?

**Comment by EsadCetiner** [comment]
@azurit I'm not, the variable is only used for static files. Do you want me to write another rule that makes that check?

**Comment by azurit** [comment]
@EsadCetiner I would better do it. I suggest to restrict it based on file extension.

**Comment by EsadCetiner** [comment]
@azurit everything good to merge now?

File: plugins/nextcloud-rule-exclusions-before.conf [hunk 1]
```diff
@@ -1124,6 +1128,16 @@ SecRule REQUEST_FILENAME "@unconditionalMatch" \
     ctl:ruleRemoveTargetById=942210;REQUEST_COOKIES:oc_sessionPassphrase,\
     ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES:oc_sessionPassphrase"
 
+# Fix false positives with static files
+SecRule REQUEST_FILENAME "@rx \.(?:avif|bmp|brotli|bvr|css|eot|gif|gz|ico|jpeg|jpg|map|m?js|mp[34]|otf|png|svg|swf|ts|ttf|txt|wav|webp|woff2?)$" \
+    "id:9508511,\
+    phase:1,\
+    pass,\
+    t:none,\
```
Comment (azurit) [inline plugins/nextcloud-rule-exclusions-before.conf:1136]: I suggest to add `t:lowercase`.

Comment (azurit) [reply plugins/nextcloud-rule-exclusions-before.conf:1136]: Or maybe better, prefix regex with `(?i)`.

**Review by azurit — APPROVED** [review]

**Pull request merged**